staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -201,6 +201,12 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume `json:"allowedFlexVolumes,omitempty" protobuf:"bytes,18,rep,name=allowedFlexVolumes"`
+	// allowedUnsafeSysctls is a list of explicitly allowed unsafe sysctls, defaults to none, * means all
+	// +optional
+	AllowedUnsafeSysctls []string `json:"allowedUnsafeSysctls,omitempty"`
```

Comment:
I see that other fields also have protobuf definitions. Is it OK that new fields don't have them?

Reply:
good catch. It's not ok.

Reply:
It's in the generated code: https://github.com/kubernetes/kubernetes/pull/63717/commits/85473f71bc15ec40417fb8c764dcb1e249ecd02a#diff-2911e6c2d742e242bdf80e7f0452650e

But again yes, it's easier to review if the generated pieces are in the first commit.

Reply:
Done.